<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartOptionsDialog</class>
 <widget class="QDialog" name="StartOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Simulation -- Options</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,0,0,0">
   <item>
    <widget class="QGroupBox" name="horizontalGroupBox_4">
     <property name="title">
      <string>Desease info</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0,1">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QCheckBox" name="immunizing_checkbox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>Immunizing desease</string>
            </property>
            <property name="tristate">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="immune_time_slider">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Time before immune turn into newly susceptible</string>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Recovery rate</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <widget class="QDoubleSpinBox" name="recovery_spinbox">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Percentage of base immune in population</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QDoubleSpinBox" name="R0_spinbox">
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QDoubleSpinBox" name="immune_spinbox"/>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>R_0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="model_type_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>SIS model</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="horizontalGroupBox_6">
        <property name="title">
         <string>Examples:</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="test_desease">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Test</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>Load Previous</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="horizontalGroupBox_3">
     <property name="title">
      <string>Population structure</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QComboBox" name="structure_combo">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <item>
           <property name="text">
            <string>Structure</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Static Grid</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Moving Grid</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Small-world Network</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Number of agents</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Number of neighbours</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QWidget" name="widget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QSpinBox" name="agents_spinbox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimum">
                  <number>100</number>
                 </property>
                 <property name="maximum">
                  <number>1000000</number>
                 </property>
                 <property name="singleStep">
                  <number>100</number>
                 </property>
                 <property name="value">
                  <number>1000</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QDoubleSpinBox" name="links_spinbox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximum">
                  <double>10.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.250000000000000</double>
                 </property>
                 <property name="value">
                  <double>4.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="structure_preview_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Starting outbreak</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="infected_slider">
       <property name="value">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="note_label">
       <property name="text">
        <string>Or</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="buttons_widget" native="true">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QPushButton" name="discrete_go_button">
          <property name="text">
           <string>Visualize Discrete Evolution</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="continuous_go_button">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Visualize Continuous Solution</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>test_desease</sender>
   <signal>pressed()</signal>
   <receiver>StartOptionsDialog</receiver>
   <slot>test_desease_pressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>structure_combo</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>StartOptionsDialog</receiver>
   <slot>structure_selected(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>14</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>infected_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_8</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>609</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>820</x>
     <y>495</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>pressed()</signal>
   <receiver>StartOptionsDialog</receiver>
   <slot>load_asked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>611</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>653</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>immunizing_checkbox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>StartOptionsDialog</receiver>
   <slot>immunizing_state_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>pressed()</signal>
   <receiver>StartOptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>584</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>discrete_go_button</sender>
   <signal>pressed()</signal>
   <receiver>StartOptionsDialog</receiver>
   <slot>discrete_go_pressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>701</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>continuous_go_button</sender>
   <signal>pressed()</signal>
   <receiver>StartOptionsDialog</receiver>
   <slot>continuous_go_pressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>740</x>
     <y>571</y>
    </hint>
    <hint type="destinationlabel">
     <x>574</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>test_desease_pressed()</slot>
  <slot>discrete_go_pressed()</slot>
  <slot>continuous_go_pressed()</slot>
  <slot>structure_selected(QString)</slot>
  <slot>load_asked()</slot>
  <slot>immunizing_state_changed(int)</slot>
 </slots>
</ui>
